src/style.css

```diff
@@ -2,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.form {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  display: block;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. You have specified a 'margin-top' here, but make sure you are not mixing 'margin-top' and 'margin-bottom' for similar elements to avoid potential margin collapse.